apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/pagerules/PageRuleServiceImpl.java

```diff
@@ -51,8 +55,10 @@ public PageRuleServiceImpl(
 
     @Override
     public CreateRuleResponse createPageRule(Long userId, CreateRuleRequest request) {
+        log.info("request.templateUid()::"+request.templateUid());
         WaRuleMetadata waRuleMetadata = setRuleDataValues(userId, request);
         log.info("Saving Rule to DB");
+        log.info("waRuleMetadata.templateUid()::"+waRuleMetadata.getWaTemplateUid());
```

Comment:
Do we still want these log messages? 

If so you can use string interpolation here instead of concatenation

`log.info("request.templateUid()::{}", request.templateUid());`